### PR TITLE
Allow token verification on get

### PR DESCRIPTION
**Pull request by mzedel**
resurrection of #149
+ test to see if the GET access is possible

### CODE REVIEW

File: api/http/api_useradm_test.go [hunk 1]
```diff
@@ -713,6 +713,20 @@ func TestUserAdmApiPostVerify(t *testing.T) {
...
+			nil)
+
+		// set these to make the middleware happy
+		req.Header.Add("X-Original-URI", "/api/mgmt/0.1/someservice/some/resource")
+		req.Header.Add("X-Original-Method", "GET")
+
+		//test
+		recorded := test.RunRequest(t, api, req)
+		mt.CheckResponse(t, tc.checker, recorded)
 	}
```
Comment (alfrunes) [inline api/http/api_useradm_test.go:730]: For the sake of coverage, maybe you could add a test case for a forwarded request as well?

File: api/http/middleware.go [hunk 2]
```diff
@@ -37,6 +37,9 @@ func ExtractResourceAction(r *rest.Request) (*authz.Action, error) {
 
 	// extract original uri
 	uri := r.Header.Get("X-Original-URI")
+	if uri == "" {
+		uri = r.Header.Get("X-Forwarded-Uri")
```
Comment (alfrunes) [inline api/http/middleware.go:41]: Cosmetic nitpick: `Uri` -> `URI`

File: api/http/api_useradm_test.go [hunk 3]
```diff
@@ -713,6 +713,20 @@ func TestUserAdmApiPostVerify(t *testing.T) {
...
+			"http://1.2.3.4/api/internal/v1/useradm/auth/verify",
+			"Bearer "+token,
+			nil)
+
+		// set these to make the middleware happy
+		req.Header.Add("X-Original-URI", "/api/mgmt/0.1/someservice/some/resource")
+		req.Header.Add("X-Original-Method", "GET")
+
+		//test
+		recorded := test.RunRequest(t, api, req)
```
Comment (alfrunes) [inline api/http/api_useradm_test.go:728]: s/:=/=/

File: api/http/middleware.go (hunk 2 above)
Comment (mzedel) [reply api/http/middleware.go]: should be fixed now 👌

File: api/http/api_useradm_test.go (hunk 3 above)
Comment (mzedel) [reply api/http/api_useradm_test.go]: should be fixed now 👌

File: api/http/api_useradm_test.go (hunk 1 above)
Comment (mzedel) [reply api/http/api_useradm_test.go:744]: should be fixed now 👌

**Pull request merged**